spec/coffee/showIn.spec.coffee

```diff
@@ -3,6 +3,11 @@ describe 'Marionette.Component', ->
     @setFixtures '<div id="main"></div>'
 
   describe 'when showing', ->
+    options = {
+      model: Backbone.Model,
+      collection: 77
```

Comment:
Instead of just testing the defaults like model and collection, let's change this to a custom property like `foo: 'bar'`.